tests/acceptance/features/apiVersions/fileVersion.feature

```diff
@@ -0,0 +1,92 @@
+    When user "Alice" gets the number of versions of file "/testfile.txt"
+    Then the HTTP status code should be "207"
+    And the number of versions should be "0"
+
+
+  Scenario: first disable versioning and create version of a file, then enabling versioning to check file version
+    Given the config "OCIS_DISABLE_VERSIONING" has been set to "true"
+    And user "Alice" has uploaded file with content "test file version 1" to "/testfile.txt"
+    And user "Alice" has uploaded file with content "test file version 2" to "/testfile.txt"
+    And the config "OCIS_DISABLE_VERSIONING" has been set to "false"
+    And user "Alice" has uploaded file with content "test file version 3" to "/testfile.txt"
+    And user "Alice" has uploaded file with content "test file version 4" to "/testfile.txt"
+    When user "Alice" gets the number of versions of file "/testfile.txt"
+    Then the HTTP status code should be "207"
+    And the number of versions should be "2"
+
+
+  Scenario: file version should not updated after versioning disabled
+    Given user "Alice" has uploaded file with content "test file version 1" to "/testfile.txt"
+    And user "Alice" has uploaded file with content "test file version 2" to "/testfile.txt"
+    When user "Alice" gets the number of versions of file "/testfile.txt"
+    Then the HTTP status code should be "207"
+    And the number of versions should be "1"
+    And the config "OCIS_DISABLE_VERSIONING" has been set to "true"
+    And user "Alice" has uploaded file with content "test file version 3" to "/testfile.txt"
+    And user "Alice" has uploaded file with content "test file version 4" to "/testfile.txt"
+    When user "Alice" gets the number of versions of file "/testfile.txt"
+    Then the HTTP status code should be "207"
+    And the number of versions should be "1"
+
+
+  Scenario Outline: check version of a file shared from project space when versioning is disabled
+    Given the config "OCIS_DISABLE_VERSIONING" has been set to "true"
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "Project1" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "Project1" with content "hello world version 1" to "text.txt"
+    And user "Alice" has uploaded a file inside space "Project1" with content "hello world version 1.1" to "text.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | text.txt |
+      | space           | Project1 |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | <role>   |
+    And user "Brian" has a share "text.txt" synced
+    When user "Alice" gets the number of versions of file "/text.txt" inside space "Project1"
+    Then the HTTP status code should be "207"
+    And the number of versions should be "0"
```

Comment:
hmm, do alice need to check the versions?

Reply:
i think version test is better this way for such scenario